docs/reference/migration/migrate_6_4.asciidoc

```diff
@@ -36,6 +36,16 @@ to follow up the operation with a request to adjust to the desired settings on
 the target index, or send the desired value of these settings with the resize
 operation.
 
+[[breaking_64_search_changes]]
+=== Search and query DSL changes
+
+==== Limiting the length of regex that can be used in a Regexp Query request
+
+<<query-dsl-regexp-query,Regexp Query>> with long string made of many operators may run into a stack overflow.
```

Comment:
Do we follow the rule `.asciidoc` files to have only 80 characters per line?

Reply:
AFAIK it's a guideline not a requirement. I can change it if it's causing formatting issues though.